website_sale_product_attachment/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from psycopg2 import sql
+
+from odoo import tools
+
+
+def migrate(cr, version):
+    """Recover name column from v12 if exists to be used as website name."""
+    column = "openupgrade_legacy_13_0_name"
+    if tools.column_exists(cr, "ir_attachment", column):
+        cr.execute(
+            sql.SQL("UPDATE ir_attachment SET website_name = {}").format(
```

Comment:
I think you're missing a `WHERE` here, to affect only website product attachments, isn't it?

Reply:
Not really, as all of them can have a name on the previous version (or at least, it will be the same as the filename), so we don't want to restrict having a value for this under a condition (like `public=True`) that can vary across time.